source/constructs/utils/utils.ts

```diff
@@ -41,3 +43,24 @@ export function addCfnCondition(resource: Resource | CfnResource | undefined, co
 
   resource.cfnOptions.condition = condition;
 }
+
+/**
+ * Run a shell command from typescript
+ * @param shellCommand Command to run in the shell
+ * @param pathToRunCommandIn CWD of shell command
+ */
+export function RunShellCommand(shellCommand: string, pathToRunCommandIn: string) {
+  execSync(shellCommand, {
+    cwd: pathToRunCommandIn,
+  });
+}
+
+/**
+ * Install dependencies of cdk project. This will include the dependencies of the
```

Comment:
Let's update a bit
```md
* Installs dependencies of the CDK project. This will include the dependencies of the
* lambda functions which must be installed before esbuild is used to create
* the bundled assets by CDK synth.
```

Reply:
Thanks for you comments. I addressed them in latest commit.